soc/riscv/riscv-privilege/gd32vf103/Kconfig.defconfig.gd32vf103

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_GD32VF103
+
+config SOC
+	default "gd32vf103"
+
+config KERNEL_ENTRY
+	default "__nuclei_start"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency) if RISCV_MACHINE_TIMER
+
+config RISCV_MACHINE_TIMER_SYSTEM_CLOCK_DIVIDER
+	default $(dt_node_int_prop_int,/soc/machine-timer@d1000000,clk-divider) if RISCV_MACHINE_TIMER
```

Comment:
Why is this not the default value directly here:
https://github.com/zephyrproject-rtos/zephyr/blob/6bb506faaf73db837891fe76433349ebfa152836/drivers/timer/Kconfig.riscv_machine#L15-L17

?

Reply:
> I think this is good so as not to break the existings.

It sure is.
Thanks for the detailed explanation, accepted.